modules/renter.go

```diff
@@ -67,9 +67,20 @@ type DownloadInfo struct {
 	StartTime   time.Time `json:"starttime"`
 }
 
+// An Allowance dictates how much the Renter is allowed to spend in a given
+// period.
+type Allowance struct {
+	Funds  types.Currency    `json:"funds"`
+	Hosts  int               `json:"hosts"`
```

Comment:
I think it makes sense to use a uint64 here, because our standard is to use uint64 anywhere that there is an unsigned number